.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/Users/sahilmalik/miniconda3/envs/p36/bin/python"
```

Comment:
This VSCode workspace settings file is a good candidate for .gitignore. You don't need it checked in.

Reply:
Its best to use native Python virtual env than miniconda. Also your case doesn't need complex package management with miniconda like ML libraries.